runtime/testy/Ansible.py

```diff
@@ -213,6 +214,7 @@ def unpackage(data):
             new_state = received_proto.student_code_status
             unpackaged_data["student_code_status"] = [new_state, time.time()]
             if self.control_state is None or new_state != self.control_state:
+                print(new_state, "UDP")
```

Comment:
It was from you guys. Let me know if i should remove it or not. 
Or you can push to this branch as well.